core/cloudflow-cli/src/main/scala/akka/cli/cloudflow/kubeclient/KubeClientFabric8.scala

```diff
@@ -212,25 +212,16 @@ class KubeClientFabric8(
 
   def getOperatorProtocolVersion(): Try[String] = withClient { client =>
     for {
-      protocolVersionCM <- Try {
+      crd <- Try {
         client
-          .configMaps()
-          .inAnyNamespace()
-          .withLabel(KubeClient.CloudflowProtocolVersionConfigMap)
-          .list()
-          .getItems()
-      }
-      protocolVersion <- {
-        protocolVersionCM.size() match {
-          case 1 => Success(protocolVersionCM.get(0))
-          case x if x > 1 =>
-            Failure(
-              CliException("Multiple Cloudflow operators detected in the cluster. This is not supported. Exiting"))
-          case x if x < 1 => Failure(CliException("No Cloudflow operators detected in the cluster. Exiting"))
-        }
+          .apiextensions()
+          .v1beta1()
+          .customResourceDefinitions()
```

Comment:
Does this need cluster admin / wide permissions?

Reply:
It's probably a good idea to have a yaml that specifies exactly which permissions CLI needs to function, similar to https://github.com/lightbend/cloudflow-helm-charts/blob/main/cloudflow/templates/02-cloudflow-operator-clusterrole.yaml, what do you think?

Reply:
that's a good idea, have to think how to produce it, maybe running the cli from within the cluster and checking permissions.